js/listdates.js

```diff
@@ -1,87 +1,58 @@
 /* global moment, humanizeDuration */
 var content = "";
```

Comment:
Last comment: If you use an IIFE to wrap your code you could `"use strict"`: https://addyosmani.com/blog/essential-js-namespacing/#beginners